lib/screens/search_screen.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+class SearchScreen extends StatefulWidget {
```

Comment:
Does search deserve an entire dedicated screen? Why is a simple Autocomplete not enough?

Reply:
Yes, we will have somewhat complex queries: languages, tags, sorting by (views, likes, dates, etc.). I believe a dedicated screen with those filtering options will provide a better UX.